src/downshift.js

```diff
@@ -661,7 +674,9 @@ class Downshift extends Component {
     this.internalSetState({
       type: Downshift.stateChangeTypes.changeInput,
       isOpen: true,
-      inputValue: event.target.value,
+      inputValue: isReactNative
+        ? /* istanbul ignore next (react-native) */ event
```

Comment:
Interesting! In react native the `event` is considered the `inputValue`? That's a surprise... Is it really a string? The `inputValue` should be a string that's the value of the input. Other parts of the code make that assumption. Just want to clarify that in react native `event` is a string that's the value of the input. If not, then we need to get the actual string value.

Reply:
Yup! This specific callback, [`onChangeText`](https://facebook.github.io/react-native/docs/textinput.html#onchangetext), takes a function that is called with the text as a string, that's it.

Other callbacks on `TextInput` that are more generic for the input itself (like `onBlur` or `onSubmitEditing`) return an event as a `Proxy` in the shape of `{ nativeEvent: { text: string } }`, however, this specific callback just gives us a `string`.